_posts/blog/2016-09-13-code-reviews.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "Handbook of manners: how to behave in a code review"
```

Comment:
Damn. Thats a complicated link and I didnt even know about capitalization in headlines at all. Thanks :)